packages/cloud-cognitive/src/components/RemoveModal/RemoveModal.js

```diff
@@ -54,6 +56,16 @@ export let RemoveModal = forwardRef(
     const primaryButtonDisabled =
       textConfirmation && userInput !== resourceName;
     const blockClass = `${pkg.prefix}--remove-modal`;
+
+    // Clear the user input this way so that if the onRequestSubmit handler fails for some reason
+    // the value of the input will still remain, we only want to empty the input value
+    // until open is false.
```

Comment:
Just a tiny suggested commentary change -- it took me a moment to work out what was happening here :-)

```suggestion
    // the value of the input will still remain: we only want to empty the input value
    // when open actually changes to false.
```